pkg/resources/custom_diffs.go

```diff
@@ -98,6 +98,21 @@ func ComputedIfAnyAttributeChanged(key string, changedAttributeKeys ...string) s
 	})
 }
 
+func ComputedIfAnyAttributeChangedWithSuppressDiff(key string, suppressDiffFunc schema.SchemaDiffSuppressFunc, changedAttributeKeys ...string) schema.CustomizeDiffFunc {
```

Comment:
(just comment, we will address this in a separate PR) the interface of this method is a bit strange, I would expect that we can pass key + its suppress diff func. Alternatively, we could just pass the keys and check their suppress diff in schema - which would allow us adding this logic to the existing `ComputedIfAnyAttributeChanged` func -> if key has suppress dif, run it, if not, just check. Please link this comment to the created issue and also add a TODO with the issue number here.

Reply:
Updated issue and linked this comment \+ added TODO comment